Assets/Scripts/NPCs/BasicNurse.cs

```diff
@@ -197,7 +197,24 @@ private void ToggleAnimations(bool toggle)
             // I was thinking that toggle=false just turns off all animations (disable the animator?)
             // and toggle=true can do specific behaivor based on currentState
             
-            animator.SetBool(Constants.ANIMATION_STEVE_MOVING, toggle);
+            if (toggle)
+            {
+                if (currentState == BasicNurseStates.Chasing)
+                {
+                    animator.SetBool(Constants.ANIMATION_BASICNURSE_RUNNING, true);
+                    animator.SetBool(Constants.ANIMATION_BASICNURSE_WALKING, false);
+                }
+                else
+                {
+                    animator.SetBool(Constants.ANIMATION_BASICNURSE_RUNNING, false);
+                    animator.SetBool(Constants.ANIMATION_BASICNURSE_WALKING, true);
+                }
+            }
+            else
+            {
+                animator.SetBool(Constants.ANIMATION_BASICNURSE_RUNNING, false);
+                animator.SetBool(Constants.ANIMATION_BASICNURSE_WALKING, false);
+            }
```

Comment:
Awesome, glad that was easy to plug right in!

Reply:
Good work sir 😉